public_html/wp-content/plugins/camptix-attendee-survey/includes/cron.php

```diff
@@ -118,6 +118,10 @@ function is_time_to_send_email( $email_id ) {
 		return false;
 	}
 
+	if ( ! 'wcpt-closed' === $wordcamp_post->post_status ) {
```

Comment:
I think here it should be `!==` instead. I've pushed a commit to make this change. If it's not right, please just revert it.

Reply:
eerg, CoPilot. Thanks Ren.